### PR TITLE
cpu: update intel cpu model types

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1857058

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
(1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5.q35: PASS (63.37 s)

cmd:
-cpu 'Cascadelake-Server-noTSX',enforce,-mpx,avx512-bf16,+kvm_pv_unhalt

**Comment by nanliu-r** [comment]
@vivianQizhu @yanan-fu @huangyum could you help to check this? thanks a lot.

File: virttest/cpu.py [hunk 1]
```diff
@@ -848,9 +848,12 @@ def _make_up_pattern(flags):
...
                                   "Opteron_G4", "Opteron_G3", "Opteron_G2",
                                   "Opteron_G1"],
-                 "GenuineIntel": ["KnightsMill", "Icelake-Server",
-                                  "Icelake-Client", "Cascadelake-Server",
-                                  "Skylake-Server", "Skylake-Client",
+                 "GenuineIntel": ["KnightsMill", "Cooperlake",
+                                  "Icelake-Server", "Icelake-Server-noTSX",
+                                  "Icelake-Client", "Icelake-Client-noTSX",
+                                  "Cascadelake-Server", "Cascadelake-Server-noTSX",
+                                  "Skylake-Server", "Skylake-Server-noTSX-IBRS",
```
Comment (huangyum) [inline virttest/cpu.py:855]: Would you pls explain why add "Skylake-Server-noTSX-IBRS" other than "Skylake-Server-noTSX"? I would prefer to keep consistent with other cpu models.

**Comment by huangyum** [comment]
BTW, CI check failed.

File: virttest/cpu.py (hunk 1 above)
Comment (nanliu-r) [reply virttest/cpu.py:855]: There is no "Skylake-Server-noTSX" model in qemu5.0, please help check again, thanks.

**Comment by nanliu-r** [comment]
> BTW, CI check failed.

Seems not my code issue.

**Comment by huangyum** [comment]
> > BTW, CI check failed.
> 
> Seems not my code issue.

Yes, it is.

/home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py:877:39: E128 continuation line under-indented for visual indent

/home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py:878:39: E128 continuation line under-indented for visual indent

/home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py:882:43: E128 continuation line under-indented for visual indent

Python library autopep8 not installed. Please install it if you want to use --fix

Style: /home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py FAIL (FIX NOT OK)

**Comment by nanliu-r** [comment]
> > > BTW, CI check failed.
> > 
> > 
> > Seems not my code issue.
> 
> Yes, it is.
> 
> /home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py:877:39: E128 continuation line under-indented for visual indent
> 
> /home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py:878:39: E128 continuation line under-indented for visual indent
> 
> /home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py:882:43: E128 continuation line under-indented for visual indent
> 
> Python library autopep8 not installed. Please install it if you want to use --fix
> 
> Style: /home/travis/build/avocado-framework/avocado-vt/virttest/cpu.py FAIL (FIX NOT OK)

Thanks and update.

**Comment by vivianQizhu** [comment]
@huangyum Any more comments? Thanks.

**Comment by vivianQizhu** [comment]
Thanks @nanliu-r @huangyum

**Pull request merged**